packages/react-native-dogfood/src/screens/Meeting/JoinMeetingScreen.tsx

```diff
@@ -42,7 +43,7 @@ const JoinMeetingScreen = (props: JoinMeetingScreenProps) => {
   );
 
   return (
-    <View style={styles.container}>
+    <SafeAreaView style={styles.container}>
```

Comment:
Was needed to be able to click on the avatar in order to logout.
The avatar was partially under the status bar which made it unclickable